operator/config/crd/bases/resources.teleport.dev_users.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I think we could remove this file (the same goes for the role's one)
CRDs are now stored in the chart itself